openedx/features/assessment/tests/test_helpers.py

```diff
@@ -0,0 +1,183 @@
+""" Test Assessment Helpers. """
+import time
+from datetime import datetime, timedelta
+
+import mock
+from ddt import data, ddt, unpack
+from django.contrib.auth.models import User
+from django.utils.timezone import now
+from opaque_keys.edx.locator import BlockUsageLocator
+from openassessment.assessment.models import Assessment, AssessmentPart
+from pytz import UTC
+
+from openedx.features.assessment import helpers
+from openedx.features.assessment.tests.factories import StudentItemFactory, SubmissionFactory
+from student.tests.factories import AnonymousUserIdFactory
+from xmodule.modulestore.django import modulestore
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+from xmodule.modulestore.tests.factories import CourseFactory, ItemFactory
+
+from .constants import (
+    COURSE_CHILD_STRUCTURE,
+    PHILU_BOT_NAME,
+    THREE_POINT_RUBRIC_DICTIONARY,
+    TWO_POINT_RUBRIC_DICTIONARY
+)
+
+FOUR_DAYS_AGO = now() - timedelta(days=4)
+
+@ddt
```

Comment:
Please confirm we need to line spaces above class

Reply:
done